pkg/ssh/ssh.go

```diff
@@ -264,7 +276,7 @@ func RunSSHUpdateMonitoringDashboards(host *models.Host, monitoringDashboardPath
 		if err := utils.FileCopy(utils.ExpandHome(customGrafanaDashboardPath), filepath.Join(monitoringDashboardPath, constants.CustomGrafanaDashboardJSON)); err != nil {
 			return err
 		}
-		if err := replaceCustomVarDashboardValues(monitoringDashboardPath, constants.CustomGrafanaDashboardJSON, chainID); err != nil {
+		if err := replaceCustomVarDashboardValues(constants.CustomGrafanaDashboardJSON, chainID); err != nil {
```

Comment:
why monitoringDashboardPath is not needed now?

Reply:
the file seems to be `filepath.Join(monitoringDashboardPath, constants.CustomGrafanaDashboardJSON)` probably it is best to now assume current working dir

Reply:
we don't need working dir anymore

Reply:
but you are correct.
Fixed